src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -177,6 +167,23 @@ const Map: React.FC<Props> = () => {
     });
   }, [identifier]);
 
+  //load landscape
+  useEffect(() => {
+    if (!landscape) {
+      get(`/api/${identifier}`).then((response) => {
+        setLandscape(response);
+        setPageTitle(response.name);
+        if (findItem) {
+          setFindFunction(() => findItem);
+        }
+      });
+
+      get(`/assessment/${identifier}`).then((response) => {
+        setAssessments(response);
+      });
+    }
+  }, [identifier, setPageTitle, setFindFunction, findItem, landscape]);
```

Comment:
Do we display any information to the user if a call to the backend is not succesful? We could add something to the `APIClient` like
````
axios.get('endpoint')
    .then((response) => {})
    .catch((error) => {
        console.log(error); //better something with information for the user
    })
````

Reply:
See https://github.com/dedica-team/nivio/issues/350